### PR TITLE
Convert kubernetes scheduler to use heron downloader.

### PR DESCRIPTION
**Pull request by cckellogg**

### CODE REVIEW

**Comment by cckellogg** [comment]
@jrcrawfo what do you think of this change?

File: heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java [hunk 1]
```diff
@@ -400,9 +404,9 @@ protected ArrayNode getPorts(ObjectMapper mapper) {
   /**
    * Get the command that will be used to retrieve the topology JAR
    */
-  protected String getFetchCommand() {
-    return "cd /opt/heron/ && curl " + Runtime.topologyPackageUri(runtime).toString()
-            + " | tar xvz";
```
Comment (jrcrawfo) [inline heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java]: No need to extract the TAR file anymore? Assuming you've tested this.

Comment (cckellogg) [inline heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java]: the downloader extracts the tar.gz on the file. There is a test for this (ExtractorTests under heron/downloaders/tests)

**Comment by jrcrawfo** [comment]
👍  looks good to me

**Pull request merged**